### PR TITLE
Update special roles

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: flask_resty_tenants/authorization.py [hunk 1]
```diff
@@ -11,10 +11,11 @@
 
 # -----------------------------------------------------------------------------
 
-NO_ACCESS = float('-inf')
+PUBLIC = float('-inf')
 READ_ONLY = 0
 MEMBER = 1
 ADMIN = 2
+NOT_ALLOWED = float('inf')
```
Comment (taion) [inline flask_resty_tenants/authorization.py:18]: `FORBIDDEN` might be a better name

Comment (itajaja) [reply flask_resty_tenants/authorization.py:18]: forbidden is not really a role tho

**Comment by taion** [comment]
@itajaja What do you think?

**Review by itajaja — APPROVED** [review]

**Comment by itajaja** [comment]
it's breaking 💥

**Pull request merged**